layouts/fastly/helix.vcl

```diff
@@ -545,6 +545,32 @@ sub hlx_type_static_url {
   call hlx_github_static_ref;
   call hlx_github_static_root;
 
+  # Only declare local variables for things we mean to change before putting
+  # them into the URL
+  declare local var.universal BOOL;   # use universal runtime?
+  declare local var.hostname STRING;  # if yes, what's the hostname
+  declare local var.namespace STRING; # namespace
+  declare local var.package STRING;   # package
+
+  # We need the action root for the next bit
+  call hlx_action_root;
+
+  if (req.http.X-Action-Root ~ "(^|^https://)([^/:\.]+)(/|\.([^/]+)/)([^/]+)") {
```

Comment:
This pattern supports both formats of the `package` parameter: `namespace/package` and `https://namespace.anywhere.run/package`